### ISSUE TITLE
[Feature]: Vue generate hooksConfig boilerplate 

### ISSUE THREAD

**Issue by sand4rt** [microsoft/playwright#31217]
### 🚀 Feature Request

Would be nice to let `create-playwright` generate the boilerplate code of the common Vue mount options. These APIs are also in Vue Test Utils and Testing Library, but many people do not know that this is also possible through the `hooksConfig`.

depends on: https://github.com/microsoft/playwright/pull/30269

### Example

```ts
// playwright/index.ts
import type {
  Component,
  DefineComponent,
  Plugin,
  ComponentOptions,
  Directive,
} from 'vue';

export type HooksConfig = {
  components?: Record<string, Component | DefineComponent>;
  plugins?: (Plugin | [Plugin, ...any[]])[];
  directives?: Record<string, Directive>;
  provide?: Record<any, any>;
  mixins?: ComponentOptions[];
};

beforeMount<HooksConfig>(async ({ app, hooksConfig }) => {
  for (const [name, component] of Object.entries(hooksConfig?.components || {}))
    app.component(name, component);

  for (const plugin of hooksConfig?.plugins || []) {
    if (Array.isArray(plugin))
      app.use(plugin[0], ...plugin.slice(1))
    else 
      app.use(plugin)
  }

  for (const [name, directive] of Object.entries(hooksConfig?.directives || {}))
    app.directive(name, directive);

  for (const [key, value] of Object.entries(hooksConfig?.provide || {}))
    app.provide(key, value);

  for (const mixin of hooksConfig?.mixins || [])
    app.mixin(mixin);
});
```

```ts
// component.test.ts
await mount(App, {
  hooksConfig: { 
    plugins: [router] 
    components: { Component },
    mixins: { Mixin },
    directives: { Directive },
  },
});
```

### Motivation

These are not documented anywhere, even though most users needs this

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you have additional information not present in this issue that you think will help prioritizing it, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
